src/api.ts

```diff
@@ -97,13 +117,22 @@ export interface Route {
   gasCostUSD?: string // Aggregation of underlying gas costs in usd
 
   containsSwitchChain?: boolean // Features required for route execution
-  infiniteApproval?: boolean // Features used for route execution
 
-  steps: LifiStep[]
+  steps: LiFiStep[]
```

Comment:
Even if small, this is a breaking change so make sure to release a major release with breaking changes in the release notes :)

Reply:
Yes, there is a bunch of breaking changes and this is one of them 😄